### PR TITLE
Extend join discussion after poll test

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

Extend join discussion after poll test by 2 weeks and add data-link-name attribute to the link to comments which should aid when querying data.

### CODE REVIEW

**Comment by GHaberis** [comment]
@gtrufitt @crifmulholland

File: static/src/javascripts/projects/common/modules/experiments/join-discussion-after-poll.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ define([
         var messageElem = document.createElement('div');
 
         messageElem.classList.add('join-discussion-after-poll');
-        messageElem.innerHTML = '<p>Thanks for taking part! Why not <a href="#comments">join the discussion</a>?</p>';
+        messageElem.innerHTML = '<p>Thanks for taking part! Why not <a data-link-name="Join discussion" href="#comments">join the discussion</a>?</p>';
```
Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/experiments/join-discussion-after-poll.js]: I would make this slightly more descriptive - `poll: cta: join discussion` so it's less likely to conflict with another event

File: static/src/javascripts/projects/common/modules/experiments/tests/join-discussion-after-poll.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ define([
     return function () {
         this.id = 'JoinDiscussionAfterPoll';
         this.start = '2016-06-15';
-        this.expiry = '2016-07-13';
+        this.expiry = '2016-07-27';
```
Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/experiments/tests/join-discussion-after-poll.js]: You also need to update your switch I imagine

Comment (GHaberis) [inline static/src/javascripts/projects/common/modules/experiments/tests/join-discussion-after-poll.js]: Ok cool, just committed that

**Comment by crifmulholland** [comment]
Hey, there's no risk on including this - so I'd increase audience. 50% seeing it (and 50% in control)  will be fine. I'd also expand 4 weeks

**Pull request merged**